### ISSUE TITLE
Few data

### ISSUE THREAD

**Issue by kimvex** [Apercu/react-native-zeroconf#40]
Hi, I have a doubt.

When I get data from a service in IOS if it returns me data and it turns out and shows me this data.

```
{ ‘webserver': 
	   { host: 'MacBook-Pro-de-Benjamin.local.',
	     txt: {},
	     addresses: 
	      [ '192.168.1.105',
	        'fe80::85:5e3d:1bb1:eae',
	        'fe80::f43a:6cff:fe12:2a97',
	        'fe80::ed0:8626:74ea:ba6c' ],
	     name: 'webserver',
	     fullName: 'MacBook-Pro-de-Benjamin.local._http._tcp.',
	     port: 3000 } }

```
In android is different in the never gets solved and only in the update event the data is shown but only this is shown.

`{ webserver: { name: 'webserver' } }`

That could be happening?

This is the code I am using.

```
      this._zeroconf = new Zeroconf();
      this._zeroconf.scan('http', 'tcp', 'local.');
      
      this._zeroconf.on('update', () => {
        console.log("UPDATE! Services are: ",this._zeroconf.getServices());
      });

      
      this._zeroconf.on('found', () => {
        console.log("Found! Services are: ",this._zeroconf.getServices());
      });

      
      this._zeroconf.on('resolved', () => {
        let d = this._zeroconf.getServices();
        this.setState({ host: d.webserver.host });
        console.log("Resolved! Services are: ",this._zeroconf.getServices());
      });      

      this._zeroconf.on('start', () => {
        console.log('Start to scan');
      });
      
      this._zeroconf.on('error', (err) => {
        console.log("An error Occurred ",err);
      });
```

**Comment by crearc** [comment]
I'm just doing Android and having the exact same issue. Basically the service was found, but never resolves. That's the same problem that this issue had: https://github.com/Apercu/react-native-zeroconf/issues/38.

**Comment by Apercu** [comment]
Might be related to #41. Do you mind trying using the `0.7.1` version?

**Comment by Apercu** [comment]
I've published a newer version, `0.8.2` that should revert to a working state. If you want to grab TXT data on Android you'll need to have an os system >= 7 though. Feel free to reopen.
